wire-ios-data-model/Tests/MLS/MLSClientManagerTests.swift

```diff
@@ -67,24 +66,33 @@ class MLSClientManagerTests: ZMBaseManagedObjectTest {
         // Given
         BackendInfo.isMLSEnabled = true
         let mlsFeature = mockFeatureRepository.fetchMLS()
-        await syncMOC.perform {
+        let domain = "example.domain.com"
+
+        let selfUser = await syncMOC.perform {
             let selfUser = ZMUser.selfUser(in: self.syncMOC)
-            selfUser.domain = "example.domain.com"
+            selfUser.domain = domain
+            return selfUser
         }
+
         let selfClient = await syncMOC.perform {
             self.createSelfClient(onMOC: self.syncMOC)
         }
+
         let hasRegisteredMLSClient = await syncMOC.perform {
             selfClient.hasRegisteredMLSClient
         }
+
         let qualifiedID = await syncMOC.perform {
-            selfClient.qualifiedClientID
+            QualifiedClientID(
+                userID: selfUser.remoteIdentifier,
+                domain: selfUser.domain ?? "",
+                clientID: selfClient.remoteIdentifier ?? "")
         }
```

Comment:
```suggestion
           let qualifiedID = try XCTUnwrap( await syncMOC.perform { selfClient.qualifiedClientID })
```
or is it really different QualifiedClientID from selfClient?

Reply:
`qualifiedClientID` property is a part of the UserClient extension in the RequestStrategy

Reply:
ah ok, I would suggest to avoid the falling back to empty string and do try XCTUnwrap instead

Reply:
@netbe It looks like `UserClient.getter:qualifiedClientID` is defined in `WireRequestStrategy`

Reply:
Using `qualifiedClientID` property from `selfClient` was the reason for importing `WireRequestStrategy`, this is wrong